app/src/main/java/com/example/fitjournal/library/presentation/screen/library/LibraryScreen.kt

```diff
@@ -35,6 +36,7 @@ fun LibraryScreen(
             ) {
                 keyboardController?.hide()
                 focusManager.clearFocus(true)
+                displayBlur.invoke(false)
```

Comment:
```suggestion
                displayBlur(false)
```

Reply:
I didi it, but in the AppScreen I saw the usage of .invoke, so when is good to use it and when is not?
is it because in the AppScreen is optional?

Reply:
if the function is nullable then you want to do function()?.invoke otherwise if we know we pass in a function then you can just call function()